src/schedule.rs

```diff
@@ -848,8 +856,8 @@ mod test {
         //let dt = schedule_tz
         //    .ymd(2019, 10, 27)
         //    .and_hms(0, 3, 29)
-        //    .checked_add_signed(chrono::Duration::hours(1)) // puts it in the middle of the DST transition
-        //    .unwrap();
+        //    .checked_add_signed(chrono::Duration::hours(1)) // puts it in the middle
+        // of the DST transition    .unwrap();
```

Comment:
This is an unfortunate wrap that might benefit from a manual adjustment.

In any case, these commented-out upstream lines are going to be removed, so we might as well not bother in this case.